dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -94,6 +95,28 @@ You can always get the latest version of the creative code below from [the AMP e
 
 {% endhighlight %}
 
+### User Sync
+
+To do User Sync on Amp pages, `amp-iframe` element need to be set up on the page as shown below. Below given `amp-iframe` will do User Sync for all the bidders in Prebid Server. We are supporting image pixel syncs as of now, iframe syncs are not supported.
```

Comment:
"To properly sync user IDs with Prebid Server, the `amp-iframe` pixel below should be added to your AMP pages.  As of now, only image pixels (those returned with `"type": "redirect"`) are supported."

Reply:
Updated